Content.Shared/Weapons/Reflect/SharedReflectSystem.cs

```diff
@@ -174,8 +174,10 @@ private void OnReflectEquipped(EntityUid uid, ReflectComponent comp, GotEquipped
         if (!TryComp(args.Equipee, out ReflectComponent? reflection))
             return;
 
-        reflection.Enabled = true;
+        if (args.Slot == "pocket1" || args.Slot == "pocket2")
```

Comment:
This should be using slot flags because this breaks as soon as someone gets 3 pockets.

Reply:
I think it doesn't require check at all, because if sword is equipped, it means that it is put in pocket, belt and so on and can't reflect. I think maybe just disabling reflecting when sword is equipped.